lib/preset-patterns.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   suit: {
     componentName: /[A-Z][a-zA-Z]+/,
     componentSelectors: suitSelector,
-    utilitySelectors: /^\.u(?:-[a-z][a-zA-Z0-9]+)+$/,
+    utilitySelectors: /^\.u(?:-[a-z][a-zA-Z0-9]*)+$/,
```

Comment:
the `+` would enforce at least two characters as the first on is covered by `[a-z]`